source/standards/operating-systems.html.md.erb

```diff
@@ -34,6 +34,6 @@ You must also make sure your OS is:
 * up to date with the latest OS releases
 * able to run on a local VM
 
-The National Cyber Security Centre (NCSC) has principles to follow for [end user device (EUD) hardening](https://www.ncsc.gov.uk/collection/end-user-device-security/eud-overview/eud-security-principles).
+The National Cyber Security Centre (NCSC) has principles to follow for [end user device (EUD) hardening](https://www.ncsc.gov.uk/collection/device-security-guidance/security-principles).
 
-The NCSC also has guidance on [hardening that's specific to `Ubuntu 18.04 LTS`](https://www.ncsc.gov.uk/collection/end-user-device-security/platform-specific-guidance/ubuntu-18-04-lts).
+The NCSC also has guidance on [hardening that's specific to `Ubuntu 18.04 LTS`](https://www.ncsc.gov.uk/collection/device-security-guidance/platform-guides/ubuntu-lts).
```

Comment:
This guidance is now specific to Ubuntu 20.04 LTS, the lint text should be updated, maybe just to:

```suggestion
The NCSC also has guidance on [hardening that's specific to the latest Ubuntu LTS version](https://www.ncsc.gov.uk/collection/device-security-guidance/platform-guides/ubuntu-lts).
```